stan/math/fwd/scal/fun/log_diff_exp.hpp

```diff
@@ -13,8 +13,11 @@ namespace math {
 template <typename T>
 inline fvar<T> log_diff_exp(const fvar<T>& x1, const fvar<T>& x2) {
   using std::exp;
```

Comment:
This wasn't introduced by this PR, but I don't see the need for the ```using std::exp;``` statements here